ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -194,6 +194,51 @@ def getQueues(siteList=None, ceList=None, ceTypeList=None, community=None, mode=
   return S_OK(resultDict)
 
 
+def getStorageElements(vo=None):
+  """
+  Get configuration of storage elements
+
+  :param str vo: select SE's for the given VO
+
+  :return: S_OK/S_ERROR, Value SE information dictionary
+  """
+
+  result = gConfig.getSections('Resources/StorageElements')
+  if not result['OK']:
+    return result
+  storageElements = result['Value']
+  baseSEs = {}
+  for se in storageElements:
+    alias = False
+    for option in ('BaseSE', 'Alias'):
+      originalSE = gConfig.getValue('Resources/StorageElements/%s/%s' % (se, option))
+      if originalSE:
+        baseSEs.setdefault(originalSE, []).append(se)
+        alias = True
+        break
+    if not alias:
```

Comment:
The `alias` variable can be removed:
```suggestion
    else:
```

Reply:
???

Reply:
I guess you're questioning what a `for..else` does?

The `else` block after a `for` loop is ran if the loop ends without `break` being explicitly used. For example:

```python
for known_user in known_users:
    if user.lower() == known_user.lower():
        print('User is already known')
        break
else:
    print('ERROR: User is not known')
```

Reply:
Wow, I have learnt something today ! Thanks